pkgs/applications/graphics/unigine-tropics/default.nix

```diff
@@ -0,0 +1,99 @@
+{ lib
+, stdenv
+, fetchurl
+, makeWrapper
+, autoPatchelfHook
+, libX11
+, libXext
+, libXrandr
+, libXinerama
+, libglvnd
+, openal
+, glibc
+, makeDesktopItem
+, imagemagick
+}:
+let
+  version = "1.3";
+
+  desktopItem = makeDesktopItem {
+    name = "Tropics";
+    exec = "Tropics";
+    genericName = "A GPU Stress test tool from the UNIGINE";
+    icon = "Tropics";
+    desktopName = "Tropics Benchmark";
+  };
+in
+stdenv.mkDerivation
+{
+  pname = "unigine-tropics";
+  inherit version;
+
+  src = fetchurl {
+    url = "http://m12-assets.unigine.com/d/Unigine_Tropics-${version}.run";
+    sha256 = "0icasdp46fjnic7gk83pknjx0gpap9j202dm0llcfg5zin5kbq7x";
+  };
+
+  libPath = lib.makeLibraryPath [
+    libglvnd
+    openal
+    glibc
+  ];
+
+  installPhase =
+    ''
+      sh $src --target $name
+
+      mkdir -p $out/lib/unigine/tropics/bin
+      mkdir -p $out/bin
+      mkdir -p $out/share/icons/hicolor
+      mkdir -p $out/share/applications/
+
+      install -m 0755 $name/bin/libUnigine_x86.so $out/lib/unigine/tropics/bin
+      install -m 0755 $name/bin/Tropics $out/lib/unigine/tropics/bin
+      install -m 0755 $name/1024x768_windowed.sh $out/bin/Tropics
+
+      cp -R $name/data $out/lib/unigine/tropics
+
+      wrapProgram $out/bin/Tropics --prefix LD_LIBRARY_PATH : $libPath:$out/lib/unigine/tropics/bin --run "cd $out/lib/unigine/tropics"
```

Comment:
```suggestion
      wrapProgram $out/bin/Tropics \
        --prefix LD_LIBRARY_PATH : $libPath:$out/lib/unigine/tropics/bin \
        --chdir $out/lib/unigine/tropics
```

Reply:
Builder called die: makeWrapper doesn't understand the arg --chdir
There is no argument chdir for makeWrapper or wrapProgram 
https://github.com/NixOS/nixpkgs/blob/master/pkgs/build-support/setup-hooks/make-wrapper.sh

Reply:
There is but it is still stuck in staging https://github.com/NixOS/nixpkgs/commit/10479e4f51ff182651ccff4a71b2a94ab93816c0